gtk/src/queuehandler.c

```diff
@@ -1456,26 +1456,11 @@ ghb_update_all_status(signal_user_data_t *ud, int status)
 static void
 save_queue_file(signal_user_data_t *ud)
 {
-    int ii, count;
-    GhbValue *queue = ghb_value_dup(ud->queue);
-
-    count = ghb_array_len(queue);
-    for (ii = 0; ii < count; ii++)
-    {
-        GhbValue *queueDict, *uiDict;
-
-        queueDict = ghb_array_get(ud->queue, ii);
-        uiDict = ghb_dict_get(queueDict, "uiSettings");
-        if (uiDict == NULL)
-            continue;
-        ghb_dict_set_int(uiDict, "job_status", GHB_QUEUE_PENDING);
-    }
-
     GtkWidget *dialog;
     GtkWindow *hb_window;
 
     hb_window = GTK_WINDOW(GHB_WIDGET(ud->builder, "hb_window"));
-    dialog = gtk_file_chooser_dialog_new("Queue Destination",
+    dialog = gtk_file_chooser_dialog_new("Export Queue",
                       hb_window,
                       GTK_FILE_CHOOSER_ACTION_SAVE,
                       GHB_STOCK_CANCEL, GTK_RESPONSE_CANCEL,
```

Comment:
First off, thanks!

PR looks mostly good.  But does not build :grimacing: 

There is a use of variable ```queue``` just below this point ```ghb_value_free(&queue);``` before queue is defined.

Reply:
ahh I see my mistake.

i'm a bit new to github collaboration, whats the correct way to submit a corrected version?

Reply:
ok, lets see if I got that right

Reply:
yup, you just commit the next change and push to your fork. The Pull request should update automatically.